vrecord

```diff
@@ -210,16 +210,24 @@ _gather_ffmpeg_vars(){
         BREW_PREFIX="/home/linuxbrew/.linuxbrew/opt/ffmpegdecklink"
     else
         BREW_PREFIX="$(brew --prefix ffmpegdecklink 2>/dev/null)"
+        if [[ -d "${BREW_PREFIX}.reinstall" ]] ; then
+            BREW_PREFIX+=".reinstall"
+        fi
     fi
     if [[ -d "${BREW_PREFIX}/bin" ]] ; then
         FFMPEG_BIN="${BREW_PREFIX}/bin/ffmpeg-dl"
-        FFPLAY_BIN="${BREW_PREFIX}/bin/ffplay-dl"
         FFPROBE_BIN="${BREW_PREFIX}/bin/ffprobe-dl"
     else
         FFMPEG_BIN="$(which ffmpeg)"
-        FFPLAY_BIN="$(which ffplay)"
         FFPROBE_BIN="$(which ffprobe)"
     fi
+
+    if [[ $("${BREW_PREFIX}/bin/ffplay-dl" ${RESOURCE_PATH}/vrecord_logo.png -t 0.01 -autoexit 2> /dev/null) ]] ; then
```

Comment:
should check `-d "${BREW_PREFIX}/bin"` first (similar to line 217). It's possible for 212 to return a null value for BREW_PREFIX

Reply:
If brew prefix returned null, wouldn't this still achieve the desired result of throwing a non zero exit status and causing `FFPLAY`_BIN to be assigned to `"$(which ffplay)"`?

Reply:
mmm, fair enough